src/components/App/App.tsx

```diff
@@ -1582,7 +1593,11 @@ export default class App extends React.Component<AppProps, AppState> {
                   <div
                     id="fullScreenContainer"
                     className={
-                      this.state.fullScreen ? 'fullScreenContainer' : ''
+                      this.state.fullScreen
+                        ? `fullScreenContainer ${
+                            !this.state.isChatEnabled ? 'noChat' : ''
```

Comment:
we probably still want to have the chat panel as video change events and system messages still output there. Just disable the chat input textbox